xapian-core/include/xapian/termgenerator.h

```diff
@@ -141,6 +144,15 @@ class XAPIAN_VISIBILITY_DEFAULT TermGenerator {
      */
     void set_stemming_strategy(stem_strategy strategy);
 
+    /** Set the stopper strategy.
+     *
+     *  @param strategy The strategy to use - possible values are:
+     *   - STOPWORDS_NONE:   
+     *   - STOPWORDS_IGNORE:   
+     *   - STOPWORDS_INDEX_UNSTEMMED_ONLY:    
+     */
+    void set_stopper_strategy(stop_strategy strategy);
```

Comment:
The behaviour of these should be documented here in the docstring. We may also be able to come up with a better name than `INDEX_UNSTEMMED_ONLY` (perhaps `SOME`, as with stemming strategy, but that might be more confusing).

We may also want to use the prefix `STOP_` instead of `STOPWORDS_`, so it's more like an instruction: "set stopping strategy to stop none". But that may involve renaming all of these enum constants to make it read sensibly.